api/standard/clause_8_ogcapi_processes.adoc

```diff
@@ -21,81 +21,90 @@ Exchange Model in GeoJSON.
 also has the effect, that the schema of the routing parameters is
 not part of the API definition, but described as a response to a
 process resource in the API (`/processes/routing`).
 
 The following table provides an overview of the resources. For details
-see the draft OGC API - Processes specification.
+see the draft _OGC API - Processes_ specification.
 
 Note that the OGC API Common requirements apply to this option, too.
 
-[#tldrwps,reftext='{table-caption} {counter:table-num}']
-.Overview of resources and applicable HTTP methods in the WPS profile
+[#tldroaproc,reftext='{table-caption} {counter:table-num}']
+.Overview of resources and applicable HTTP methods in the _OGC API - Processes_ profile
 [cols="32,25,10,33",options="header"]
 !===
 |Resource |Path |HTTP method |Document reference
-|Landing page |`/` |GET |<<wps_landing_page>>
-|Conformance declaration |`/conformance` |GET |<<wps_conformance>>
-|Processes |`/processes` |GET |<<wps_processes>>
-|Process |`/processes/routing` |GET |<<wps_process>>
-.2+|Jobs .2+|`/processes/routing/jobs` |GET |<<wps_jobs>>
-|POST |<<wps_new_job>>
-|Job |`/processes/routing/jobs/{jobId}` |GET |<<wps_job_status>>
-|Result |`/processes/routing/jobs/{jobId}/result` |GET |<<wps_result>>
+|Landing page |`/` |GET |<<oaproc_landing_page>>
+|Conformance declaration |`/conformance` |GET |<<oaproc_conformance>>
+|Processes |`/processes` |GET |<<oaproc_processes>>
+|Process |`/processes/routing` |GET |<<oaproc_process>>
+.2+|Jobs .2+|`/jobs` |GET |<<oaproc_jobs>>
+|POST |<<oaproc_new_job>>
+|Job |`/jobs/{jobId}` |GET |<<oaproc_job_status>>
+|Result |`/jobs/{jobId}/results` |GET |<<oaproc_result>>
 !===
 
-[[wps_landing_page]]
+NOTE: Proposal being discussed ( https://github.com/opengeospatial/ogcapi-processes/issues/124 ) to POST execution request to `/processes/{processId}` instead of to `/jobs`.
+If we manage to fully harmonize the Processes / `/routes` approach, `/routes` could be an alias to `/processes/routing` supporting same POST requests.
+
+[[oaproc_landing_page]]
 === Landing page
 
 See <<landing_page>>, except that the "data" link references `/processes`.
 
-[[wps_conformance]]
+[[oaproc_conformance]]
 === Conformance declaration
 
-The following is an example of the conformance declaration for the WPS option
-that implements all requirements classes. Note that the following classes
-of the Routing API option are not available in the WPS option as OGC API Processes
-does not support the required capabilities:
+The following is an example of the conformance declaration for the _OGC API - Processes_ option
+that implements all requirements classes. The ResultSet class has no equivalent functionality with _OGC API - Processes_.
 
-* Delete route
-* Callback
-* Result set
-* Synchronous execution
+Suggestion: Define a single specific conformance class for the OGC API - Processes variants, and re-use the existing Routing conformance classes?
```

Comment:
This may not be easy, but we can try. I think we want to keep the wording of the normative statements as simple/understandable as possible, but this may be a challenge, if the requirements would have to be written to apply to both /routes and /processes/routing, potentially with slightly different payloads.

We should remove this here and convert it into an issue.

Reply:
Moved to https://github.com/opengeospatial/ogcapi-routes/issues/15#issuecomment-784356194